libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -4,7 +4,7 @@
  */
 #pragma once
 #include <eosio/chain/types.hpp>
-
+#include <eosio/chain/config.hpp>
```

Comment:
I don't see anything new in this file that would require a new include, is it needed, or was this accidentally brought in?